### DISCUSSION TITLE
UCB CS70: discrete Math and probability theory - CSDIY.wiki

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#379] [Announcements]
# UCB CS70: discrete Math and probability theory - CSDIY.wiki

CS自学指南

https://csdiy.wiki/en/%E6%95%B0%E5%AD%A6%E8%BF%9B%E9%98%B6/CS70/

**Comment by zivmax** [comment]
The lectures on the official website are unavailable now; Is there any replacement?

**Reply by w-lambda-6** [reply]
2012:https://www.youtube.com/watch?v=AAtXWGwxI9k
2015:https://www.bilibili.com/video/BV1gf4y1q7UY
